src/main/java/co/cask/gcp/bigquery/BigQuerySinkConfig.java

```diff
@@ -39,10 +41,12 @@
   @Macro
   public String table;
 
+  @Nullable
   @Name("bucket")
   @Description("The Google Cloud Storage bucket to store temporary data in. "
     + "It will be automatically created if it does not exist, but will not be automatically deleted. "
-    + "Cloud Storage data will be deleted after it is loaded into BigQuery.")
+    + "Cloud Storage data will be deleted after it is loaded into BigQuery." +
+    "If it is not provided, a UUID will be used as bucket name and will be deleted after the run is finished.")
```

Comment:
put the + on this line for consistency.

Reply:
If it is not provided, a unique bucket will be created and then deleted after the run finishes.